packages/kit/src/runtime/client/client.js

```diff
@@ -1459,12 +1459,15 @@ export function create_client({ target, base }) {
 					event.submitter
 				);
 
-				const method = submitter?.formMethod || form.method;
+				const method =
+					submitter?.formMethod ||
+					/** @type {HTMLFormElement} */ (HTMLFormElement.prototype.cloneNode.call(form)).method;
 
 				if (method !== 'get') return;
 
 				const url = new URL(
-					(submitter?.hasAttribute('formaction') && submitter?.formAction) || form.action
+					(submitter?.hasAttribute('formaction') && submitter?.formAction) ||
+						/** @type {HTMLFormElement} */ (HTMLFormElement.prototype.cloneNode.call(form)).action
```

Comment:
I don't think we need these changes, they can be reverted. `form` is already a cloned node.

Reply:
Whoops, you're right! Didn't notice that the form has already been cloned. That's negligence on my part. I thought to change all `form.method` and `form.action` references before just because of the possible override that happens without cloning. Changed it.